### ISSUE TITLE
Topic 7, Lesson 2: Test ID 7.B, text "and" vs "or"

### ISSUE THREAD

**Issue by Ailikekaus** [Section508Coordinators/DHSA-TT-200-A#173]
## Issue Type
- [ ] System Bug (the course doesn't function properly)
- [x] Typo/grammar
- [ ] Incorrect content
- [ ] 508 issue (something is not accessible according to our own test process)
- [ ] Other, specify...[replace this text with your issue type description]

## Description
Test ID 7.B Test requirement text states: `role="presentation"` _**and**_ `aria-hidden="true"` while all other references in the training and test script are _**or**_.

Makes it sound like an image needs both attributes `role="presentation"` and `aria-hidden="true"` to be decorative.

## Location
https://training.section508testing.net/mod/book/view.php?id=1238&chapterid=2833
Topic 7
Lesson 2: Decorative Images
Test ID 7.B
Test requirement text

## Actual Behavior / Observation
Text states: `role="presentation"` _**and**_ `aria-hidden="true"` while all other references in the training and test script are _**or**_.
>>The purpose of this test is to validate that each decorative image has no accessible name or description as indicated in ANDI Output; a null/empty accessible name and description and other attributes, such as role="presentation" _**and**_ aria-hidden="true", will alert assistive technology (AT) to ignore the image and avoid unnecessary content being read out loud.

![image](https://user-images.githubusercontent.com/19952958/59954626-ad2b4000-9453-11e9-9aa5-c13d23a78e0c.png)

https://training.section508testing.net/mod/book/view.php?id=1238&chapterid=2837
Topic 7
Lesson 2: Decorative Images
Test ID 7.B
Evaluating content
>>ANDI Output is empty for an image that has role="presentation" _**or**_ aria-hidden="true". These are two techniques to hide an image from AT and are appropriate for decorative content.

![image](https://user-images.githubusercontent.com/19952958/59954700-0d21e680-9454-11e9-8113-35b412bd9ebb.png)

## Expected Behavior / Recommendation
Update text from "and" to "or".

## Steps to Reproduce the Issue (if applicable)
N/A

**Comment by AnnMarieDavis** [comment]
Text changed to recommended edit (text is not bold in the course): ...such as role="presentation" **or** "aria-hidden="true"....
